modules/backend/behaviors/RelationController.php

```diff
@@ -1096,7 +1096,9 @@ public function onRelationManageCreate()
 
             $modelsToSave = $this->prepareModelsToSave($newModel, $saveData);
             foreach ($modelsToSave as $modelToSave) {
-                $modelToSave->save(null, $this->manageWidget->getSessionKey());
+                if (get_class($modelToSave) !== get_class($newModel)) {
```

Comment:
Does $newModel always get saved in `$this->relationObject->add($newModel, $sessionKey)`? And does the add method work if the newModel object doesn't exist yet?

Reply:
@LukeTowers As far as I can tell, yes, the model is always saved in `$this->relationObject->save` - unless it's a `belongsTo(Many)` relation. Does the relation manager support that type of relation? The save occurs whether deferred binding is enabled or not.

This change is only for the "create" portion of the Relation manager, so `$newModel` is always effectively a new instance of the model that the relation manager is tied to. If you edit or delete the relation, this happens in different methods, and don't seem to exhibit the same doubling-up of the save events.